src/map/utils/synthutils.cpp

```diff
@@ -492,32 +476,31 @@ uint8 calcSynthResult(CCharEntity* PChar)
     return result;
 }
 
-/************************************************************************
-*                                                                       *
-*  Пытаемся увеличить умение персонажа.                                 *
-*  Desynthesis (разбор предметов на запчасть) не увеличивает умение.    *
-*                                                                       *
-*  Ломать не строить. Ломание столов и стульев не сделает из нас        *
-*  плотника, значит умение в данном случае не повышается.               *
-*                                                                       *
-************************************************************************/
+/********************************************************************
+*                                                                   *
+* We are trying to increase the skill of the character.             *
+* Desynthesis does not increase skill. NOT TRUE, RATE IS JUST LOWER *
+* Breaking does not build. Breaking tables and chairs won't make us *
+* carpenter, then skill in this case does not increase.             *
```

Comment:
Is this currently being reflected in the code? If not, the comment can be removed - we don't need to keep the inaccurate comments 😉